### PR TITLE
chore(format): update coding style

### PR DESCRIPTION
**Pull request by appleboy**
#### Summary

Just update the coding style.

1. `var for` => `foo :=`
2. Adjacent parameters with the same type should be grouped together

### CODE REVIEW

File: server/services/store/mockstore/mockstore.go [hunk 1]
```diff
@@ -5,9 +5,10 @@
 package mockstore
 
 import (
+	reflect "reflect"
```
Comment (jespino) [inline server/services/store/mockstore/mockstore.go:8]: This is a generated file, it shouldn't be modified manually.

File: server/services/store/sqlstore/initializations/bindata.go [hunk 2]
```diff
@@ -53,18 +53,23 @@ type bindataFileInfo struct {
 func (fi bindataFileInfo) Name() string {
 	return fi.name
 }
+
```
Comment (jespino) [inline server/services/store/sqlstore/initializations/bindata.go:56]: This file is generated so it shouldn't be modified manually.

File: server/services/store/sqlstore/migrations/postgres/bindata.go [hunk 3]
```diff
@@ -63,18 +63,23 @@ type bindataFileInfo struct {
 func (fi bindataFileInfo) Name() string {
 	return fi.name
 }
+
```
Comment (jespino) [inline server/services/store/sqlstore/migrations/postgres/bindata.go:66]: This file is generated so it shouldn't be modified manually.

File: server/services/store/sqlstore/migrations/sqlite/bindata.go [hunk 4]
```diff
@@ -63,18 +63,23 @@ type bindataFileInfo struct {
 func (fi bindataFileInfo) Name() string {
 	return fi.name
 }
+
```
Comment (jespino) [inline server/services/store/sqlstore/migrations/sqlite/bindata.go:66]: This file is generated so it shouldn't be modified manually.

File: server/services/telemetry/mocks/ServerIface.go [hunk 5]
```diff
@@ -76,7 +76,7 @@ func (_m *ServerIface) GetRoleByName(_a0 string) (*model.Role, *model.AppError)
 }
 
 // GetSchemes provides a mock function with given fields: _a0, _a1, _a2
-func (_m *ServerIface) GetSchemes(_a0 string, _a1 int, _a2 int) ([]*model.Scheme, *model.AppError) {
+func (_m *ServerIface) GetSchemes(_a0 string, _a1, _a2 int) ([]*model.Scheme, *model.AppError) {
```
Comment (jespino) [inline server/services/telemetry/mocks/ServerIface.go:79]: This file is generated so it shouldn't be modified manually.

**Review by jespino — CHANGES_REQUESTED** [review]
Everything looks good to me except for the modifications done in automatically generated files, if you can revert them, would be great. Thanks @appleboy!

**Comment by appleboy** [comment]
@jespino Thanks for your feedback. All done in 2b4e03e.

**Review by jespino — APPROVED** [review]
Looks good to me. Thanks @appleboy! :confetti_ball:

**Pull request merged**